src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePicker.Flyout.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using Uno.UI;
+
+namespace Windows.UI.Xaml.Controls
+{
+	partial class DatePicker
+	{
+
+
+#if __IOS__ || __ANDROID__
+		public static DependencyProperty UseNativeStyleProperty { get; }= DependencyProperty.Register(
+			"UseNativeStyle",
+			typeof(bool),
+			typeof(DatePicker),
+			new PropertyMetadata(!FeatureConfiguration.Style.UseUWPDefaultStyles));
```

Comment:
Use `!Style.ShouldUseUWPDefaultStyle(typeof(DatePicker))` instead - this will also take `FeatureConfiguration.Style.UseUWPDefaultStylesOverride` into account

Even then, this approach sets up a non-obvious race between modifying the FeatureFlag and calling the `DatePicker` static constructor...

Reply:
I makes no sense to change it globally during the execution of an app.  And if you really want to do that, you can using a `<Style />`.

Reply:
Fixed everything.  In my next push.